content-scripts/inject/run-userscript.js

```diff
@@ -1,7 +1,10 @@
 import Addon from "../../addon-api/content-script/Addon.js";
+const ranAddons = [];
```

Comment:
This logic should likely not be in this file, which simply exports a function

Reply:
I think it needs to...